compiler/src/dotty/tools/dotc/typer/Implicits.scala

```diff
@@ -683,6 +776,33 @@ trait Implicits { self: Typer =>
     }
   }
 
+  /** An addendum to an error message where the error might be fixed
+   *  be some implicit value of type `pt` that is however not found.
+   *  The addendum suggests suggests implicit imports that might fix the problem.
+   */
+  override def implicitSuggestionsFor(pt: Type)(given ctx: Context): String =
```

Comment:
In fact an empty string is preferable, since it's easier to consume. Generally, the compiler always prefers sentinels over optional values, as long as feasible.